api/views/phones.py

```diff
@@ -509,6 +510,17 @@ def voice_status(request):
     relay_number, _ = _get_phone_objects(called)
     relay_number.remaining_seconds = relay_number.remaining_seconds - int(call_duration)
     relay_number.save()
+    if relay_number.remaining_seconds < 0:
+        profile = relay_number.user.profile_set.first()
+        info_logger.info(
+            "phone_limit_exceeded",
+            extra={
+                "fxa_uid": profile.fxa.uid,
+                "relay_number_enabled": relay_number.enabled,
+                "remaining_seconds": relay_number.remaining_seconds,
+                "remaining_minutes": relay_number.remaining_minutes,
```

Comment:
Please include the `call_duration` as well.

Reply:
Completed in b9e544d